nrich-notification/README.md

```diff
@@ -33,33 +46,76 @@ To be able to use this module following configuration is required:
-`ObjectErrors`. Default implementation is `MessageSourceNotificationMessageResolverService` that resolves messages from Spring's `MessageSource`.
+### [NotificationMessageResolverService][nrich-notification-message-resolver-service-url]
 
-`NotificationResolverService` is main service that is responsible for creating notifications. It can create notifications for validation errors, exceptions or action names.
-Consequently, it accepts either Spring's `Errors` or Jakarta `ConstraintViolationException` for creating validation notifications, `Throwable` for creating error notifications and
-`String` for action notification.
+[`NotificationMessageResolverService`][nrich-notification-message-resolver-service-url] is a service responsible for resolving messages from message codes and `ObjectErrors`.
+Default implementation is [`MessageSourceNotificationMessageResolverService`][message-source-notification-message-resolver-service-url] that resolves messages from Spring's `MessageSource`.
 
-`NotificationResponseService` is a service that adds support for returning data with notifications by introducing additional methods that return `NotificationDataResponse`.
+---
 
-When using Spring Web MVC users can use `NotificationResponseService` implementation `WebMvcNotificationResponseService` that is a wrapper around `NotificationResolverService` providing a
-convenient way of returning a notification with a response.
+### [NotificationResolverService][notification-resolver-service-url]
+
+[`NotificationResolverService`][notification-resolver-service-url] is the core service that is responsible for creating notifications.
+It can create notifications for validation errors, exceptions or action names.
+Consequently, it accepts either Spring's `Errors` or Jakarta `ConstraintViolationException` for creating validation notifications, `Throwable` for creating error notifications and `String` for action
+notification.
+
+---
+
+### [NotificationResponseService][notification-response-service-url]
+
+[`NotificationResponseService`][notification-response-service-url] is a service that adds support for additional data in notifications by introducing additional methods that
+return [`NotificationDataResponse`][notification-data-response-url].
+When using Spring Web MVC users can use [`NotificationResponseService`][notification-response-service-url] implementation
+[`WebMvcNotificationResponseService`][web-mvc-notification-response-service-url] that is a wrapper around [`NotificationResolverService`][notification-resolver-service-url] providing a convenient way
+of returning a notification with a response.
+
+---
+
+### Default `MessageSource` setup
+
+`nrich-notification` also includes [`nrich-notification-messages`][nrich-notification-messages-url] property files that contain required default messages and should be added to
+`MessageSource` (**this is done automatically when using Spring Boot starter module**).
```

Comment:
I don't think we need this in bold, especially if it's inside parentheses.

Additionally, not sure about `and should be added to MessageSource` part. It sounds a bit strange to me.

Reply:
Agreed.